cygnus-ngsi/src/main/java/com/telefonica/iot/cygnus/utils/NGSIUtilsForTests.java

```diff
@@ -94,7 +96,24 @@ public static Context createContextForOrion(String orionHost, String orionPort,
         context.put("orion_fiware_path", orionFiwarePath);
         return context;
     } // createContextForOrion
-    
+
+    /**
+     * Creates a Flume context for Arcgis sinks.
+     * 
+     * @param url
+     * @param username
+     * @param password
+     * @return
+     */
+    public static Context createContextForArcGis(String url, String username,
```

Comment:
PR should include also a unit test to cover the new functionality. I see this addition to NGSIUtilsForTest so I guess that was your idea, but the PR doesn't include any test at the end...

Reply:
Added NGSIArcGisSinkTest